ui/apps/platform/cypress/integration/violations/violations.test.js

```diff
@@ -274,7 +274,8 @@ describe('Violations', () => {
         );
     });
 
-    it('should show a resolved violation in the details page', () => {
+    // TODO(ROX-27198): The test is failing since 2024-11-27.
+    it.skip('should show a resolved violation in the details page', () => {
```

Comment:
I would keep the test name the same because we are collecting that information, and by changing the name, it would be identified as a new test.

Also all other occurrences of `it.skip` do not use that pattern. In most cases, there is only (a non-standardized) comment before it.